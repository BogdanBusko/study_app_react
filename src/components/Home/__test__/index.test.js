import React from 'react';

import { customRender } from '../../../test_helpers/helpers';
import Home from '../index';

it("renders correctly", () => {
  const { asFragment } = customRender(<Home />);

  expect(asFragment(<Home />)).toMatchSnapshot();
});
