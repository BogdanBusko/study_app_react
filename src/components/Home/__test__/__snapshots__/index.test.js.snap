// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Home component
    </h1>
  </div>
</DocumentFragment>
`;
